packages/manager/modules/account-sidebar/src/controller.js

```diff
@@ -26,11 +26,17 @@ export default class OvhManagerAccountSidebarCtrl {
       this.sidebarExpand = false;
     });
 
-    const unregisterListener = this.$rootScope.$on('ovh-chatbot:enable', () => {
-      this.hasChatbot = true;
-      this.links = this.getLinks();
-      unregisterListener();
-    });
+    const unregisterListener = this.$rootScope.$on(
+      'ovh-chatbot:enable',
+      (event, hasChatbot, showLivechat) => {
+        this.hasChatbot = true;
+        this.links = this.getLinks();
+        unregisterListener();
+        if (this.hasChatbot && showLivechat) {
```

Comment:
As we update `this.hasChatbot` with `true` 3 lines before, maybe we can remove the `this.hasChatbot` from the condition ?